SECURITY.md

```diff
@@ -0,0 +1,11 @@
+## Report a security issue
+
+The tokio project team welcomes security reports and is committed to providing prompt attention to security issues. Security issues should be reported privately via [security@tokio.rs](mailto:security@tokio.rs).
```

Comment:
```suggestion
The Tokio project team welcomes security reports and is committed to providing prompt attention to security issues. Security issues should be reported privately via [security@tokio.rs](mailto:security@tokio.rs).
```

Reply:
Should we explicitly discourage reporting security issues on the public issue tracker, here?

Reply:
Should we have a definition of what qualifies as a security issue? As a library, I think there is some grey area around _vulnerabilities which exist in Tokio_ "out of the box", without requiring users to take specific action, versus APIs which _allow users to write potentially vulnerable code_. I think it would be good to clarify what issues should be reported privately, and for which security advisories will be issued.

Reply:
Good question! The scope of the reporting policy is a judgement call; if you have a sense of what kind of security issues would be worthwhile for tokio to handle I'd encourage you to add it (either now or later).

IME it is difficult to strictly define what kind of security issues should be reported (outside of directed efforts), since security issues can inherently be surprising by nature. For example, an API which allows a user to write vulnerable code may be worthwhile if many users have misused the API. My advice would be to start broadly, and if triage or coordination becomes cumbersome, tune the policy accordingly.

I like the the two scenarios you called out; I'll plan to add them as examples to the text -- please feel free to reject/edit at will.